flash/vision/backbones.py

```diff
@@ -140,3 +147,21 @@ def torchvision_backbone_and_num_features(model_name: str, pretrained: bool = Tr
         return backbone, num_features
 
     raise ValueError(f"{model_name} is not supported yet.")
+
+
+def timm_backbone_and_num_features(model_name: str, pretrained: bool = True) -> Tuple[nn.Module, int]:
```

Comment:
Please, add a test.

Reply:
[This test](https://github.com/aniketmaurya/lightning-flash/blob/issue_172-add_timm/flash/vision/backbones.py#L155) I have added.
Could you please refer me to where do I have to add the test for this function?

Reply:
Do I have to add a backbone name for timm in the list? @aribornstein

Reply:
@tchaton what do you prefer?

Reply:
I have added a test for it.